<ui version="4.0" >
 <author>Ricardo Villalba</author>
 <class>FilePropertiesDialog</class>
 <widget class="QDialog" name="FilePropertiesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>SMPlayer - File properties</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="info_page" >
      <attribute name="title" >
       <string>&amp;Information</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>6</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QTextEdit" name="info_edit" >
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="demuxer_page" >
      <attribute name="title" >
       <string>&amp;Demuxer</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>6</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="textLabel2" >
         <property name="text" >
          <string>&amp;Select the demuxer that will be used for this file:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
         <property name="buddy" >
          <cstring>demuxer_listbox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="demuxer_listbox" />
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>241</width>
             <height>41</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="resetDemuxerButton" >
           <property name="text" >
            <string>&amp;Reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="vc_page" >
      <attribute name="title" >
       <string>&amp;Video codec</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>6</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="textLabel3" >
         <property name="text" >
          <string>&amp;Select the video codec:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
         <property name="buddy" >
          <cstring>vc_listbox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="vc_listbox" />
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>241</width>
             <height>41</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="resetVCButton" >
           <property name="text" >
            <string>&amp;Reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ac_page" >
      <attribute name="title" >
       <string>A&amp;udio codec</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>6</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="textLabel4" >
         <property name="text" >
          <string>&amp;Select the audio codec:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
         <property name="buddy" >
          <cstring>ac_listbox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="ac_listbox" />
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>241</width>
             <height>41</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="resetACButton" >
           <property name="text" >
            <string>&amp;Reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="options_page" >
      <attribute name="title" >
       <string>&amp;MPlayer options</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Additional Options for MPlayer</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel3_2" >
            <property name="text" >
             <string>Here you can pass extra options to MPlayer.
Write them separated by spaces.
Example: -flip -nosound</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="textLabel1_3" >
              <property name="text" >
               <string>&amp;Options:</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
              <property name="buddy" >
               <cstring>mplayer_args_edit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="mplayer_args_edit" />
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line2" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape" >
             <enum>QFrame::HLine</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="textLabel4_2" >
            <property name="text" >
             <string>You can also pass additional video filters.
Separate them with ",". Do not use spaces!
Example: scale=512:-2,eq2=1.1</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="textLabel2_3" >
              <property name="text" >
               <string>V&amp;ideo filters:</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
              <property name="buddy" >
               <cstring>mplayer_vfilters_edit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="mplayer_vfilters_edit" />
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line3" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape" >
             <enum>QFrame::HLine</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="textLabel5_2" >
            <property name="text" >
             <string>And finally audio filters. Same rule as for video filters.
Example: resample=44100:0:0,volnorm</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="textLabel3_3" >
              <property name="text" >
               <string>Audio &amp;filters:</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
              <property name="buddy" >
               <cstring>mplayer_afilters_edit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="mplayer_afilters_edit" />
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>resetDemuxerButton</tabstop>
  <tabstop>resetVCButton</tabstop>
  <tabstop>resetACButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FilePropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>267</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel" >
     <x>267</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FilePropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>267</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel" >
     <x>267</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
